tube.js

```diff
@@ -97,9 +100,15 @@ export class Tube {
         if (this.debug) console.log("host write " + utils.hexword(addr) + " = " + utils.hexbyte(b));
         switch (addr & 7) {
             case 0:
-                if (b & 0x80) this.r1stat |= b & 0x3f;
-                else this.r1stat &= ~(b & 0x3f);
-                this.hstat[0] = (this.hstat[0] & 0xc0) | (b & 0x3f);
+                if (b & 0x80)
+                this.r1stat |= b & 0x3f;
+                else if (this.r1stat & 0x20) {
+                    this.reset();
+                    this.parasiteCpu.reset();
+                }
+                else
+                    this.r1stat &= ~(b & 0x3f);
+                    this.hstat[0] = (this.hstat[0] & 0xc0) | (b & 0x3f);
```

Comment:
I _think_ you're either:

* missing an undent here OR
* you're missing `{` and `}` around lines 110 and 111

Either way, if you tested this and it worked then unindenting is what's needed.

Reply:
I think we should use {} for all if/else statements (otherwise someone will misunderstand the code in 20 years and mess it up 😁)

Reply:
It was the former. Have adjusted the line spacing/indentation.

I agree with { } for all if/else statements. I've put them in around this part.